server/config-loader.ts

```diff
@@ -33,6 +37,12 @@ export function load(invalidate = false) {
 		}
 	}
 
+	for (const [preset, values] of FLAG_PRESETS) {
+		if (process.argv.includes(preset)) {
+			for (const value of values) config[value] = true;
+		}
+	}
```

Comment:
How is the port detected? That's probably relevant if you're not going to remove the settings from `process.argv`.

Reply:
I'd rather just remove the flag from process.argv and let port be handled normally, if I'm understanding you correctly?

Reply:
a9b4303 should do it.

Reply:
I don't like how you do the removal and detection in separate loops. It's easy enough just to skip everything starting with `-` when detecting the port.

Reply:
There, does that work better?